parsl/executors/high_throughput/manager_selector.py

```diff
@@ -0,0 +1,22 @@
+import copy
+import random
+from typing import Dict, List
+
+from parsl.executors.high_throughput.manager_record import ManagerRecord
+
+
+class ManagerSelectorBase:
+
+    def sort_managers(self, ready_managers: Dict[bytes, ManagerRecord], manager_list: List[bytes]) -> List[bytes]:
+        raise NotImplementedError
+
+
+class ManagerSelectorRandom(ManagerSelectorBase):
+
+    def __init__(self):
```

Comment:
maybe you can omit this __init__ entirely?

Reply:
Will do